cmd/bridge/config.go

```diff
@@ -70,8 +70,9 @@ func SetFlagsFromConfig(fs *flag.FlagSet, filename string) (err error) {
 		return err
 	}
 
-	if config.APIVersion != "console.openshift.io/v1beta1" || config.Kind != "ConsoleConfig" {
-		return fmt.Errorf("unsupported version (apiVersion: %s, kind: %s), only console.openshift.io/v1beta1 ConsoleConfig is supported", config.APIVersion, config.Kind)
+	if !((config.APIVersion == "console.openshift.io/v1beta1" && config.Kind == "ConsoleConfig") ||
+		(config.APIVersion == "config.openshift.io/v1" && config.Kind == "Console")) {
```

Comment:
This is a completely different API resource. We should not accept it here.

Reply:
`config.openshift.io/v1` `Console` does not have everything we need to start the console, nor do we want it to. It's a small subset of the options that we want to expose as a cluster operator config.

Reply:
Right, I'm suggesting it should stay `ConsoleConfig`, but we prob want to get rid of the `beta`, as we have moved everything else out of `beta`.  Correct?